src/frontend.rs

```diff
@@ -31,16 +31,31 @@ impl std::fmt::Display for CompileError
 	}
 }
 
-pub fn compile_ron_definition(input_string : &str, options_opt : Option<CompileOptions>) -> Result<String, CompileError>
+fn read_definition(input_string : &str, assembly : bool) -> Result<Definition, CompileError> {
```

Comment:
This function inherits my complaints about a boolean for specifying which language to use